drivers/espi/espi_taf_npcx.c

```diff
@@ -444,6 +452,69 @@ static int espi_taf_npcx_flash_erase(const struct device *dev, struct espi_saf_p
 	return 0;
 }
 
+#ifdef CONFIG_ESPI_TAF_NPCX_RPMC_SUPPORT
+static int espi_taf_npcx_rpmc_op1(const struct device *dev, struct espi_saf_packet *pckt)
+{
+	struct espi_taf_npcx_pckt *taf_data_ptr = (struct espi_taf_npcx_pckt *)pckt->buf;
+	uint8_t *data_ptr = (uint8_t *)(taf_data_ptr->data);
```

Comment:
isn't data a `uint8_t *` alrady?

```c
struct espi_taf_npcx_pckt {
        uint8_t tag;
        uint8_t *data;
};  
````

Reply:
Done. The data is already a ```uint8_t *```. Have removed this casting.